hooks/hookexecution/context.go

```diff
@@ -0,0 +1,71 @@
+package hookexecution
+
+import (
+	"sync"
+
+	"github.com/golang/glog"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/hooks/hookstage"
+)
+
+// executionContext holds information passed to module's hook during hook execution.
+type executionContext struct {
+	endpoint       string
+	stage          string
+	accountId      string
+	account        *config.Account
+	moduleContexts *moduleContexts
+}
+
+func (ctx executionContext) getModuleContext(moduleName string) hookstage.ModuleInvocationContext {
+	moduleInvocationCtx := hookstage.ModuleInvocationContext{Endpoint: ctx.endpoint}
+	if ctx.moduleContexts != nil {
+		if mc, ok := ctx.moduleContexts.get(moduleName); ok {
+			moduleInvocationCtx.ModuleContext = mc
```

Comment:
Is it possible to update tests to cover this line?

Reply:
The purpose of this logic is for communication between different stages, so that module can pass some data to itself from earlier stages to later ones. Right now we have implementation for `entrypoint` stage only, but we will definitely cover this logic with implementations of new stages.